pkgs/applications/radio/inspectrum/default.nix

```
@@ -19,7 +21,7 @@ stdenv.mkDerivation {
     sha256 = "1fvnr8gca25i6s9mg9b2hyqs0zzr4jicw13mimc9dhrgxklrr1yv";
   };
 
-  nativeBuildInputs = [ pkgconfig ];
+  nativeBuildInputs = [ pkgconfig wrapQtAppsHook ];
```

```suggestion
  nativeBuildInputs = [ pkgconfig ];
```

is automatically added